dist/index.mjs

```diff
@@ -13,16 +13,21 @@ class Applications {
             endpoint: `/projects/${options.projectId}/applications?${params.toString()}`,
         });
     }
-    async get(applicationId) {
+    async get(applicationId, isCatalog) {
+        const endpoint = isCatalog
+            ? `/catalog/${applicationId}`
+            : `/applications/${applicationId}`;
         return this.base.request({
             method: "GET",
-            endpoint: `/applications/${applicationId}`,
+            endpoint,
         });
     }
-    async list(projectId) {
+    async list(projectId, isCatalog) {
```

Comment:
let's just fetch catalog applications if the first argument is the string "catalog"

otherwise a user has to pass `applications.list(undefined, true)` in order to fetch catalog application vs `applications.list("catalog")`

```typescript
    async list(projectId: string | null = null): Promise<Application[]> {
        let endpoint: string;
        switch (projectId) {
            case null:
                endpoint = "/applications";
                break;
            case "catalog":
                endpoint = "/catalog";
                break;
            default:
                endpoint = `/projects/${projectId}/applications`;
        }

        return this.base.request<Application[]>({
            method: "GET",
            endpoint,
        });
    }
```